### PR TITLE
PCQ-727: Update to add consolidation service AAT kustomise config

### PR DESCRIPTION
**Pull request by davejones74**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PCQ-727

### Change description ###
Update to change pipeline to use kustomise.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/pcq/pcq-consolidation-service/pcq-consolidation-service.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-consolidation-service
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:prod-*
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-consolidation-service/pcq-consolidation-service.yaml:8]:
```suggestion
  annotations:
    hmcts.github.com/prod-automated: enabled
```

File: k8s/namespaces/pcq/pcq-consolidation-service/aat-01.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: pcq-consolidation-service
+spec:
+  values:
+    job:
+      schedule: "0/45 * * * *"
+    global:
+      environment: aat
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-consolidation-service/aat-01.yaml:13]:
```suggestion
 
```

File: k8s/namespaces/pcq/pcq-consolidation-service/aat-00.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: pcq-consolidation-service
+spec:
+  values:
+    job:
+      schedule: "0/15 * * * *"
+    global:
+      environment: aat
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-consolidation-service/aat-00.yaml:13]:
```suggestion
  
```

**Review by adusumillipraveen — COMMENTED** [review]
Added some suggestions to simplify the config. Also, you should remove the raw yamls in common/ directory (it  will fail if not )

File: k8s/namespaces/pcq/pcq-consolidation-service/pcq-consolidation-service.yaml (hunk 1 above)
Comment (davejones74) [reply k8s/namespaces/pcq/pcq-consolidation-service/pcq-consolidation-service.yaml]: Thanks @adusumillipraveen your suggestions have been implemented and I've removed the common equivilents.

**Review by adusumillipraveen — COMMENTED** [review]
You need to delete this as well https://github.com/hmcts/cnp-flux-config/blob/e5465ccd7a09d6b2b84deeb27aec962db37bdb57/k8s/aat/cluster-00/pcq/pcq-consolidation-service.yaml

also, its missing enties like rd have :
 https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/cluster-00-overlay/kustomization.yaml#L5
 
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/cluster-00-overlay/rd/kustomization.yaml

**Comment by davejones74** [comment]
Thanks @adusumillipraveen the first file you indicated the cluster configuration was deleted. I've added the missing overlay entries to point back to the namespace configuration.

File: k8s/aat/cluster-00-overlay/rd/kustomization.yaml [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
-kind: Kustomization
-namespace: rd
-bases:
-- ../../../namespaces/rd/rd-judicial-data-load/rd-judicial-data-load.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/rd-location-ref-data-load.yaml
-patchesStrategicMerge:
-- ../../../namespaces/rd/rd-judicial-data-load/aat-00.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/aat-00.yaml
\ No newline at end of file
+    kind: Kustomization
```
Comment (adusumillipraveen) [inline k8s/aat/cluster-00-overlay/rd/kustomization.yaml:2]: can you revert these pls

File: k8s/namespaces/pcq/kustomization.yaml [hunk 5]
```diff
@@ -4,3 +4,4 @@ namespace: pcq
 bases:
 - namespace.yaml
 # Warning : Adding a file here, adds to all environments to which you add your kustomization.
+- pcq-consolidation-service/pcq-consolidation-service.yaml
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/kustomization.yaml:7]: you shouldn't add this
```suggestion

```

File: k8s/aat/cluster-00-overlay/rd/kustomization.yaml (hunk 4 above)
Comment (davejones74) [reply k8s/aat/cluster-00-overlay/rd/kustomization.yaml]: Thanks blaming Inteliji on that one, hopefully this is better now!

**Comment by davejones74** [comment]
The files changed is not picking up that I deleted  k8s/aat/cluster-00/pcq/pcq-consolidation-service.yaml which is strange but if you look on the branch it's definately gone.

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**